README.md

```diff
@@ -70,6 +70,7 @@ config :sentry,
 | `hackney_opts` | False  | `[pool: :sentry_pool]` | |
 | `hackney_pool_max_connections` | False  | 50 | |
 | `hackney_pool_timeout` | False  | 5000 | |
+| `pre_event_send_function` | False | | |
```

Comment:
From our conversation in #137, it sounded like you agreed with my suggestion of `before_send_event` for this instead of this name.  Any reason you decided to use this name after all?